api/proto/config/devconfig.proto

```diff
@@ -80,6 +80,9 @@ message EdgeDevConfig {
   // not start applications etc as to enable inspection of its state and
   // recover from bad state.
   bool maintenance_mode = 24;
+  // epoch indicates current epoch of config
```

Comment:
I'd suggest renaming this to controller_epoch with the comment saying it captures when a new controller takes over and needs all the info be resent.